pages/common/enca.md

```diff
@@ -0,0 +1,19 @@
+# enca
+
+> Detect and convert encoding of text files
+
+- detect file encoding; -L option tell enca the current language; LANGUAGE as zh_CN, en_US etc.
```

Comment:
Just to be clear, I didn't suggest to remove the examples, the only part I was suggesting to replace was the word "as" :)